cpp/ql/src/Likely Bugs/Likely Typos/FutileParams.ql

```diff
@@ -1,22 +0,0 @@
-/**
- * @name Non-empty call to function declared without parameters
- * @description A call to a function declared without parameters has arguments, which may indicate
- *              that the code does not follow the author's intent.
- * @kind problem
- * @problem.severity warning
- * @precision very-high
- * @id cpp/futile-params
- * @tags correctness
- *       maintainability
- */
-
-import cpp
-
-from Function f, FunctionCall fc
-where fc.getTarget() = f
-  and f.getNumberOfParameters() = 0
-  and not f.isVarargs()
-  and fc.getNumberOfArguments() != 0
-  and not f instanceof BuiltInFunction
-  and exists(FunctionDeclarationEntry fde | fde = f.getADeclarationEntry() | not fde.isImplicit())
-select fc, "This call has arguments, but $@ is not declared with any parameters.", f, f.toString()
```

Comment:
You will need to add a change note so that any customers who are using this query can figure out what they will need to run instead.

Reply:
... ah, it looks like the query is new and may not have been in any releases yet.  In that case, all that is needed is a change note for the new queries.

Reply:
This file is no longer used; look at <code>cpp/ql/src/Likely Bugs/UnderspecifiedFunctions/*.ql</code> instead.  Sorry for moving things around like that, I just wanted to create a more modular folder structure.

Reply:
By changelog, do you mean <code>*.qhelp</code>.  Also, what do you mean by "what they will need to run instead"?

Reply:
The change notes are here: https://github.com/Semmle/ql/tree/master/change-notes

A change note is required for any change that might affect customers, which includes moving an existing query.

Reply:
I looked at the changelogs at the URL you provided, but don't see any references to query locations (by which I'm assuming you mean locations of the <code>.ql</code> files).  I do see references to query IDs, though...